dotcom-rendering/src/model/window-guardian.ts

```diff
@@ -31,6 +29,37 @@ export interface WindowGuardianConfig {
 	};
 }
 
+interface WindowGuardianFrontConfig {
```

Comment:
bikedshed: Do we have a rule of when to use types or interfaces? Just asking as this looks like like a type subjectively.

Reply:
They are more or less interchangeable so I tend not to think too much about it. The only difference I'm aware of is an interface can be extended so perhaps feels more... *-struggles for word-* ... systemy?